build/gen-release-patch.sh

```diff
@@ -76,10 +76,28 @@ update_capabilities() {
     git add --intent-to-add capabilities/v$VERSION.json
 }
 
+update_versioned_docs() {
+    local versions_dir="docs/versions"
+    local version_docs_dir="${versions_dir}/v$VERSION"
+
+    local major=$(cut -d. -f1 <<<$VERSION)
+    local minor=$(cut -d. -f2 <<<$VERSION)
+    shopt -s nullglob # no literal string when glob does not match
+    for prev in ${versions_dir}/v${major}.${minor}.*; do
+        rm -r $prev
+        ln -sf v$VERSION $prev
```

Comment:
This is a blunt approach. Even if it might work, redirects (`x.y.{0..n-1} -> x.y.n` if `n` is the latest patch version) seem better.

Reply:
Due to the way Hugo seems to do symlinks -- it seems to not create two copies -- we can't do this.